docker-compose.yml

```diff
@@ -4,20 +4,27 @@ services:
   asa_server:
     tty: true
     container_name: asa_server
+    restart: unless-stopped
     build:
       context: .
     env_file:
       - .env
     volumes:
-      - ./ShooterGame:/opt/arkserver/ShooterGame
-      - /opt/steamcmd
+      - shooter_game:/opt/arkserver
+      - steam_data:/opt/steamcmd
     ports:
       # Port for connections from ARK game client
       - "${SERVER_PORT}:${SERVER_PORT}/udp"
+      #uncomment if you want to expose the RCON Port
+      #- "${RCON_PORT}:${RCON_PORT}/tcp"
     networks:
       - default
     deploy:
       resources:
         limits:
           memory: 16g
     memswap_limit: 16g
+
+volumes:
+    shooter_game:
```

Comment:
So ./ShooterGame:/opt/arkserver?

Reply:
I'm not quite sure why exactly you want that change to happen. If we only save the /opt/arkuser/ShooterGame path, the server will redownload the game everytime the container gets re-build e.g. by changes in the .env or smth like that. 

I mean we could additionally save the /opt/arkserver/Shootergame in the ./ShooterGame value?

Reply:
My bad, you're absolutely right. What do you think of putting the ./ShooterGame folder inside an ark-data folder. And then we use a default mount of ./ark-data:/opt/arkserver?
This way, we prevent re-downloading part of the game *and* we ensure that the settings/saves can be easily changed. If you're okay with that change, I'll merge the PR once this is done.

Reply:
I'm not quite sure, but when implementing your change and using ./ark_data as a local volume to bind /opt/arkserver the Dockerfile is not able to copy the start.sh script anymore to /opt/arkserver, even when granting chmod 777 on the folder.. Using an external volume everything works fine.

Copying the start.sh script anywhere outside that volume works, but I'm not sure if thats a valid workaround.

Reply:
Let's use this workaround for now, I'll try to find a more elegant solution in the future for the volumes. As mentioned in #5, it would be nice to split volumes between saves, game storage and steam temporary downloads.

Reply:
I'm really not getting behind this. Just mounting - ./ark_data:/opt/arkserver creates the folder+volume in the repo path, but docker still isn't granted permissions to write in this volume, so it simply won't work.

Do you know any workarounds for this? (Other than granting permission manually)

Reply:
I think we have to create the folder manually and grant permissions to user 1000:1000. There's currently no other workaround that I'm aware of unfortunately. It's either that or running the container as root always which I'm not really comfortable with. I dm'ed you on discord, it'll be faster if we have to discuss further.